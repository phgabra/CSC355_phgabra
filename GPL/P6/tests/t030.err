Semantic error on line 1: The given expression is not a legal type for the size of array <r1>.  The size must be an integer.  Given expression is of type string.
Semantic error on line 2: The given expression is not a legal type for the size of array <r2>.  The size must be an integer.  Given expression is of type double.
Semantic error on line 3: The array 'r3' was declared with illegal size '-2'. Arrays sizes must be integers of 1 or larger.
Semantic error on line 4: The array 'r4' was declared with illegal size '0'. Arrays sizes must be integers of 1 or larger.
