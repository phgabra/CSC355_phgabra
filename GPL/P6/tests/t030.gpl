rectangle r1["hello"];
rectangle r2[2.2];
rectangle r3[40 - 42];
rectangle r4[0];
