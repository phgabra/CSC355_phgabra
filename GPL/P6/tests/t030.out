gpl.cpp::main()
  input file(tests/t030.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

4 errors found.
gpl giving up.
